docs/usage/security-modes.md

```diff
@@ -0,0 +1,52 @@
+# Security modes
+
+This section describes the different **Security mode** options for connectivity that can be selected for Liqo, regulating connectivity restriction for intra-cluster traffic.
+
+## Overview
+
+Actually, Liqo provides two security modes:
+
+* **Full pod-to-pod connectivity**: represents the native behaviour of Liqo, there are no restriction in terms of connectivity, and all pods of the first cluster can connect to all pods of the second cluster (and vice versa).
+* **Intra-cluster traffic segregation**: full pod-to-pod connectivity is enabled only within real clusters; regarding intra-cluster traffic, local cluster has the possibility to contact only its offloaded pods, while remote cluster is allowed to contact exclusively services offloaded on it
+
+## Selection of the Security mode
+
+The desired Security mode can be selected with a **feature flag** at install time (see the [reference](/installation/install.md)):
+
+* `--set networking.securityMode=FullPodToPod` to select **full pod-to-pod connectivity (default mode)**
+* `--set networking.securityMode=IntraClusterTrafficSegregation` to select **intra-cluster traffic segregation**
+
+```{admonition} Note
+Currently, the selection of security mode is possible only at installation time. In future, this feature will be extended to be configured at run-time, e.g., each time you setup a new peering.
+```
+
+## Connectivity matrix
+
+To better explain security modes, schemas of possible Liqo topologies and relative traffic matrices are provided in the following.
+<details>
+<summary>Full pod-to-pod connectivity</summary>
+<IMG src="/_static/images/usage/security-modes/full_p2p.svg"/>
+</details>
+
+<details>
+<summary>Intra-cluster traffic segregation</summary>
```

Comment:
If you can, use Markdown syntax instead of HTML. If markdown doesn't work, please leave a comment.